src/JWTToken.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Picqer\BolRetailerV8;
+
+class JWTToken
```

Comment:
De naam `JWTToken` is een beetje redundant, dat is een beetje hetzelfde als "ATM machine" of "PIN number". JWT staat voor JSON Web Token, dus `JWTToken` is nu JSON Web Token Token 😄